### DISCUSSION TITLE
Sometimes, closing a tab with a Webview triggers a closure of another such tab

### DISCUSSION THREAD

**Discussion by ivan-demchenko** [microsoft/vscode-discussions#709] [Extension Development QnA] [answered]
Hi all,

I am observing this weird behaviour with Webviews. Sometimes, if I've got a few Webview-tabs opened, and if I close one of them using the keyboard, another one gets closed too. I noticed this issue in some other extensions too. I have to admit, the apps that get loaded in the webviews are not trivial, but they're also not very heavy.

Has anybody noticed a similar issue? If so, how did you overcome it?

Related issue: https://github.com/microsoft/vscode/issues/188669

**Comment by r3m0t** [comment, marked as answer]
I think you should use "Developer: Toggle Keyboard Shortcuts Troubleshooting" to create a detailed log and open an issue on the microsoft/vscode repo on Github. Ideally w/ a publicly available extension.

**Reply by ivan-demchenko** [reply]
Thank you @r3m0t,
Your suggestion helped me to discover an interesting behaviour. Indeed, the `workbench.action.closeActiveEditor` command sometimes gets invoked twice, and the second time is due to `triggered via menu entry accelerator`. This helps to formulate a solid foundation for the GH issue 👍
